packages/surveys/src/lib/recall.ts

```
@@ -55,7 +55,7 @@ export const parseRecallInformation = (
   variables: TResponseVariables
 ) => {
   const modifiedQuestion = structuredClone(question);
-  if (question.headline && question.headline[languageCode]?.includes("recall:")) {
+  if (question.headline && question.headline[languageCode].includes("recall:")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)